accepted/2020/workloads/workload-manifest.md

```diff
@@ -176,15 +178,25 @@ Workload definitions take the following form:
 | `kind` | string | Either `build` or `dev`. Default is `dev`. | No |
 | `description` | string | User-visible description for the workload. | Yes if dev and non-abstract |
 | `packs` | string array | IDs of the packs that are included in the workload. | No |
-| `extends` | string array | IDs of workloads whose packs should be included in this workload. | No |
-| `platforms` | string array | Limits the workload and workloads that extend it to only be shown and installed on these host platforms. The strings are RIDs. | No |
+| `extends` | string array | IDs of "base" workloads whose packs should be included in this workload. | No |
+| `platforms` | string array | Restricts the workload and workloads that extend it to only be shown and installed on these host platforms. Inherits any restrictions imposed by base workloads. The strings are RIDs. | No |
```

Comment:
does this only support portable RIDs or the full RID graph? e.g. `osx-arm64` vs. `osx.11.0-arm64`

Reply:
It uses the chain in `sdk/[VERSION]/NETCoreSdkRuntimeIdentifierChain.txt`